test/test_models.py

```diff
@@ -49,3 +51,6 @@ def test_waveform(self):
         out = model(x)
 
         assert out.size() == (batch_size, output_dims, num_features - pad * 2)
+
+if __name__ == '__main__':
+    unittest.main()
```

Comment:
nit/very subjective: although it does not hurt but I personally find adding `unittest.main` redundant and unnecessary.

Reply:
:) since the other tests have it, let's keep that line in for consistency with the other files and other domains (e.g. [here](https://github.com/pytorch/vision/blob/master/test/test_datasets.py)), but feel free to open a pull request to discuss removing them all

Reply:
FYI: It was added to the other files without review or approval in https://github.com/pytorch/audio/pull/509, and it is not used by any official automated jobs. So it is not very convincing to keep this for the sake of consistency.

Reply:
since we don't want to block this PR on this, feel free to open a separate PR to change this :)

Reply:
@jimchen90 -- can you move this change to a separate pull request? thanks!

Reply:
Thanks, do you mean that I open a separate pull request without these two lines after this file is merged?

Reply:
Yes. Thanks